output-data/changelogs/tag_tree.json

```diff
@@ -0,0 +1,72 @@
+{
+    "Other": {},
+    "Item": {
+        "Weapon Item": {},
+        "Vitality Item": {},
+        "Spirit Item": {}
+    },
+    "Shop": {},
+    "Souls": {
+        "Soul Orb": {},
+        "Sinner's Sacrifice": {},
+        "Soul Urn": {}
+    },
+    "Hero": {
+        "Ability": {},
+        "HeroLab Calico": {},
+        "HeroLab Fathom": {},
+        "HeroLab Holliday": {},
+        "HeroLab Magician": {},
+        "HeroLab Viper": {}
+    },
+    "Powerup": {},
+    "Breakable": {
+        "Crate": {},
+        "Golden Statue": {}
+    },
+    "Zipline": {},
+    "Objective": {
+        "Guardian": {},
+        "Base Guardian": {},
+        "Walker": {},
+        "Patron": {
+            "Weakened Patron": {}
+        },
+        "Weakened Patron": {},
+        "Shrine": {}
+    },
+    "Patron": {
+        "Weakened Patron": {}
+    },
+    "NPC": {
+        "Denizen": {},
+        "Creep": {
+            "Denizen": {},
+            "Trooper": {}
```

Comment:
trooper appears in three places. There should be a clear hierarchy

Reply:
Looks like there is two separate issues here:

The first being that Trooper is under NPC when it only needs to be under Creep (a child of NPC).

The second being that:
```
"Creep": {
        "Denizen": {},
        "Trooper": {}
}
 ```
Is appearing both at the root level, and under NPC. So I'll need to look into fixing that, nice catch!